YAXLib/Customization/ITypeContext.cs

```diff
@@ -42,7 +42,7 @@ public interface ITypeContext
     /// <param name="obj">The object of type <see cref="Type"/> to serialize.</param>
     /// <param name="options">The <see cref="SerializerOptions"/> or <see langword="null"/> to take options from the parent <seealso cref="YAXSerializer"/>.</param>
     /// <returns>The <see cref="XElement"/> representation of the object graph.</returns>
-    XElement Serialize(object obj, SerializerOptions options = null);
+    XElement Serialize(object obj, SerializerOptions options);
```

Comment:
So the options are now required?  

I think keeping it optional is easier to use. (We need an overload then)